unity-environment/Assets/ML-Agents/Scripts/ExternalCommunicator.cs

```diff
@@ -240,25 +256,21 @@ public void giveBrainInfo(Brain brain)
     {
         string brainName = brain.gameObject.name;
```

Comment:
we try to use C#'s type inference as much as we can. For instance:

`string brainName = "blah";`
becomes
`var brainName = "blah";`

it might seem trivial, but it helps *alot* during code refactoring. and makes you write C# code much faster :)
Also, when the returning value is a complicated generic, it's much nicer to do:

`var theStates = brain.states;`
than
`IEnumerable<KeyValuePair<int,int>> brain.state;`